py3status/modules/dropboxd_status.py

```diff
@@ -86,14 +86,16 @@ def dropbox(self):
 
         if status == "Dropbox isn't running!":
             color = self.py3.COLOR_BAD
-            status = self.status_off
+            _format = self.status_off
         elif status == "Up to date":
             color = self.py3.COLOR_GOOD
-            status = self.status_on
+            _format = self.status_on
         else:
             color = self.py3.COLOR_DEGRADED
             if self.status_busy is not None:
-                status = self.status_busy
+                _format = self.status_busy
+
+        status = self.py3.safe_format(_format, {'status': status})
```

Comment:
> I still have to pass in {'status': status} twice. 

but it means different things and is also not documented.  The new defaults in that branch I don't like either the explicit were nicer.

you can just do a push -f to this branch to remove my update